lambdas/credentialissuerconfig/build.gradle

```diff
@@ -24,7 +24,8 @@ dependencies {
 			"org.mockito:mockito-core:4.1.0",
 			"org.mockito:mockito-junit-jupiter:4.1.0",
 			"com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.13.0",
-			"com.nimbusds:oauth2-oidc-sdk:9.3.1"
+			"com.nimbusds:oauth2-oidc-sdk:9.3.1",
+			project(":lib").sourceSets.test.output
```

Comment:
Just out of interest, what is this doing?

Reply:
There's a class in the lib projects test directory called `TestFixtures` which has got a bunch of things like private/public keys and pre-signed verifiable credentials. Stuff that you might need that can be a pain to set up for individual tests.
As it's in the lib projects tests it wasn't on the class path for the lambda projects tests. This adds the lib projects test source set to the credenentialissuerconfig lambda projects test dependencies, which makes it accessible.

Reply:
So it allows me to do this: https://github.com/alphagov/di-ipv-core-back/pull/176/files#diff-8720e48a70c9d82f7e6ef03d4dd907c0d36c87975f30d1a45b239ed2d6cd56a1R26

Reply:
Makes sense, thanks for explaining the gradle wizardry.